pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -425,6 +424,18 @@ func indexField(field string, maxindex int) []string {
 	for i := 2; i <= maxindex; i++ {
 		fieldIndexes = append(fieldIndexes, field[0:i])
```

Comment:
I wonder how does this work with UTF-8 🤔 e.g. for `DisplayName`?

Reply:
Good question...I suspect this should convert to runes.

Reply:
I added new functionality and tests to ensure that we are "correctly" normalising strings that we are searching.